server/jetstream_cluster.go

```diff
@@ -4797,6 +4790,13 @@ func (cc *jetStreamCluster) selectPeerGroup(r int, cluster string, cfg *StreamCo
 			continue
 		}
 
+		// If ignore skip
+		if ip != nil {
```

Comment:
nil maps ok to dereference. So you can just do line 4795 direct.

https://go.dev/play/p/XLBsPw7NUBM

Reply:
This is where you can just dereference, and if nil will be ok.